torch/utils/tensorboard/x2num.py

```diff
@@ -0,0 +1,34 @@
+# DO NOT alter/distruct/free input object !
```

Comment:
destruct

I also don't understand what this comment is saying

Reply:
This is a note for converting objects from other frameworks. Removed since it is now pytorch only.